nixos/modules/programs/chromium.nix

```diff
@@ -69,11 +69,22 @@ in
       extraOpts = mkOption {
         type = types.attrs;
         description = ''
-          Extra chromium policy options, see
-          <link xlink:href="https://www.chromium.org/administrators/policy-list-3">https://www.chromium.org/administrators/policy-list-3</link>
-          for a list of avalible options
+          Extra chromium policy options, see available options at 
+          <link xlink:href="https://cloud.google.com/docs/chrome-enterprise/policies/">https://cloud.google.com/docs/chrome-enterprise/policies/</link>.
```

Comment:
I was intrigued that the link goes to information about chrome, not chromium. Maybe explain in a comment?

Reply:
good idea! check the latest version

i also added a comment about the policies that would have saved me an hour of debugging